packages/orbit-design-tokens/output/tokens.less

```diff
@@ -272,11 +271,11 @@
 @alert-color-text-warning: rgb(173, 87, 0);
 @alert-color-text-link: rgb(79, 94, 113);
 @border-radius-badge: 12;
-@badge-bundle-basic-background: linear-gradient(to top right, #E13E3B 0%, #E87E09 100%);
+@badge-bundle-basic-background: linear-gradient(to top right, #e13e3b 0%, #e87e09 100%);
```

Comment:
This is, again, with that problem of formatting on save. Please revert to avoid unnecessary diffs in the future (same for other output files, eventually)